backend/app/api/routes/projects.py

```
@@ -259,89 +259,89 @@ async def generate_description_document(
 
 
 @router.post("/{project_id}/step2")
-async def generate_architecture_document(
+async def generate_standards_document(
     project_id: str,
     input_data: dict,
```

Ruff (0.13.1)

Unused function argument: `input_data`

(ARG001)